### PR TITLE
Add documentation related to python-esiclient cluster commands

### PR DESCRIPTION
**Pull request by tzumainn**
This documentation adds a cluster provisioning section, and updates the OpenShift section with information regarding how to use python-esiclient for orchestration.

### CODE REVIEW

**Comment by tzumainn** [comment]
@DanNiESh hi! could you take a look at this documentation update when you have the time?

File: docs/source/usage/openshift.rst [hunk 1]
```diff
@@ -150,14 +147,77 @@ Add Hosts
...
+.. prompt::
+
+  {
+      "cluster_name": "name-of-cluster",
+      "openshift_version": "4.13.12",
+      "high_availability_mode": "Full",
+      "base_dns_domain": "your.dns",
+      "api_vip": "aaa.aa.aa.aaa",                  // a free IP on your private subnet
+      "ingress_vip": "bbb.bb.bb.bbb",              // a free IP on your private subnet
+      "ssh_public_key": "public-key",              // the actual key (and not just the path to the key)
```
[the remark this answers is not in the document]
Comment (tzumainn) [reply docs/source/usage/openshift.rst]: Ah, yes, exactly! My wording was bad, but 'content' was exactly the word I was looking for - thanks!

File: docs/source/usage/openshift.rst [hunk 2]
```diff
@@ -150,14 +147,77 @@ Add Hosts
...
+Orchestrate the Cluster
+~~~~~~~~~~~~~~~~~~~~~~~
+
++---------------------+-----------------------------------------------------------------------+
+|                     | **Actions**                                                           |
++---------------------+-----------------------------------------------------------------------+
+| Orchestrate Cluster | ``openstack esi openshift orchestrate <path-to-cluster-config-file>`` |
++---------------------+-----------------------------------------------------------------------+
+
+Once the cluster config file is created, run ``refresh-token`` and then ``openstack esi cluster orchestrate`` to start the orchestration of the OpenShift cluster.
```
[the remark this answers is not in the document]
Comment (tzumainn) [reply docs/source/usage/openshift.rst]: Oh, definitely - thanks for catching this! Fixed.

**Comment by tzumainn** [comment]
Thanks for the review! Let me know what you think about the updates.

File: docs/source/usage/cluster.rst [hunk 3]
```diff
@@ -0,0 +1,148 @@
+Installing a Cluster on ESI
+===========================
+
+A lessee wishing to create a cluster out of multiple leased nodes and a private network can do so by running the correct `CLI commands`_. Alternatively, they can use the cluster CLI commands in `python-esiclient`_ to simplify the workflow involved in creating and managing their clusters.
```
Comment (DanNiESh) [inline docs/source/usage/cluster.rst:4]: I'm not quite clear about the relationship between `openstack esi cluster orchestrate` and openstack esi openshift orchestrate`

Comment (tzumainn) [reply docs/source/usage/cluster.rst:4]: Oh, they're completely independent.

**Pull request merged**